cves/CVE-2017-9787.yml

```diff
@@ -127,7 +134,12 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
+  answer: I could not find anything relating to the bug's discovery. All I found was
```

Comment:
Did you try searching bug reporting forums such as the Struts Jira?(https://issues.apache.org/jira/projects/WW/issues)

Searching for "Yasser Zamani Apache Struts" on Google or in Jira might also reveal something about why you can't find how it was discovered (such as if they are part of some internal team or something).

Reply:
I tried what you suggested and was able to find a couple of dates at least.